DESCRIPTION

```diff
@@ -36,6 +37,7 @@ URL: https://github.com/McDowellLab/loadflex
 Collate:
```

Comment:
collate should not be created directly, but roxygen2 creates if for you if you have files that depend on other files (you specify with `@include`, e.g., here https://github.com/USGS-R/loadflex/blob/master/R/loadModel.R#L10)